hazelcast/src/test/java/com/hazelcast/test/HazelcastTestSupport.java

```diff
@@ -735,6 +736,48 @@ public static void assertOpenEventually(String message, CountDownLatch latch, lo
         }
     }
 
+    public static void assertCountEventually(final String message, final CountDownLatch latch, final int expectedCount, long timeoutInSeconds) {
+        assertTrueEventually(new AssertTask() {
+            @Override
+            public void run() throws Exception {
+                for (int i = 0; i < 2; i++) { // recheck to see if hasn't changed
+                    if (latch.getCount() != expectedCount) {
+                        throw new AssertionError("Latch count has not been met. " + message);
+                    }
+                    sleepMillis(50);
+                }
+            }
+        }, timeoutInSeconds);
+    }
+
+    public static void assertAtomicEventually(final String message, final AtomicInteger atomic, final int expectedValue, int timeoutInSeconds) {
+        assertTrueEventually(new AssertTask() {
+            @Override
+            public void run() throws Exception {
+                for (int i = 0; i < 2; i++) { // recheck to see if hasn't changed
+                    if (atomic.get() != expectedValue) {
+                        throw new AssertionError("Atomic value has not been met. " + message);
+                    }
+                    sleepMillis(50);
+                }
+            }
+        }, timeoutInSeconds);
+    }
+
+    public static void assertAtomicEventually(final String message, final AtomicBoolean atomic, final boolean expectedValue, int timeoutInSeconds) {
```

Comment:
just a minor one - the expected value is traditionally before the actual value in an argument list.